# yaw desired and pitch desired rotation 
float64 HeadYaw
float64 HeadPitch
---
# movement result
bool result